pyfilesync/console.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Program makes backup of your file
+Example usage
+=============
+
+To invoke program type in your console::
+
+    filesync --help
+
+"""
+
+import logging
+from argparse import ArgumentParser
+from pyfilesync import get_version
+
+
+def __option_args(argv=None):
```

Comment:
Argumenty aplikacji konsolowej.

Ta funkcja ma za zadanie sparsować parametry które będą podawane podczas wywoływania programu. Robi to obiekt `ArgumentParser`. Metody `add_argument()` dodają nowe parametry które będą obsługiwane przez aplikację. Tutaj są opisane dwa parametry:

 * `-v` lub `--version` - Wypisuje Numer wersji programu (w tym przypadku bierze wersję biblioteki)
 * `-l` lub `--log` z parametrem ustawia poziom logowania. 
 * Opcja `--help` jest generowana automatycznie, dlatego nie ma jej w kodzie.

W samej funkcji macie kilka parametrów do ustawienia jak `dest`, który określa w jakiej zmiennej będzie informacja z parametru, `help` - gdzie dajecie opis do `--helpa` czy `action` gdzie określacie co się stanie jeśli opcja będzie podana

Więcej przeczytacie w dokumentacji `ArgumentParsera`

Jeśli chodzi o logowanie to używam biblioteki `logging` o której też polecam poczytać ;)